Makefile.common

```diff
@@ -224,6 +225,12 @@ DISTNAME := scummvm-$(DISTVERSION)
 DISTDIR := dist
 VERFILE := $(DISTDIR)/$(DISTNAME)/base/internal_version.h
 
+#ifdef USE_PANDOC
```

Comment:
Do you really mean to comment this line? I suspect you want `ifdef USE_PANDOC` and not `#ifdef USE_PANDOC` (and same for the endif below)?

As it is when pandoc is not present on the system, of if `--enable-pandoc` is not used when running configure, it fails because it tries to do this but `PANDOC` is not defined.

Reply:
Yep, left that over when I was debugging...